### ISSUE TITLE
使用PPOCRv3进行训练时，启动过会就自动停了，没有任何提示

### ISSUE THREAD

**Issue by asus10tu** [PaddlePaddle/PaddleOCR#6754]
- 系统环境/System Environment：win10
- 版本号/Version：Paddle：paddlepaddle-gpu==2.2.2  PaddleOCR： paddleocr==2.5.0.3
- 运行指令/Command Code：python tools/train.py -c .\configs\rec\PP-OCRv3\ch_PP-OCRv3_rec_distillation.yml
- 完整报错/Complete Error Message：

![zzzzz](https://user-images.githubusercontent.com/13192980/176657390-ac060846-4257-4aa6-aa46-f537dcddc740.png)

内存和显存均没有溢出的情况下，突然就中断了
以下为日志：

[2022/06/30 18:34:27] ppocr INFO: Architecture : 
[2022/06/30 18:34:27] ppocr INFO:     Models : 
[2022/06/30 18:34:27] ppocr INFO:         Student : 
[2022/06/30 18:34:27] ppocr INFO:             Backbone : 
[2022/06/30 18:34:27] ppocr INFO:                 last_conv_stride : [1, 2]
[2022/06/30 18:34:27] ppocr INFO:                 last_pool_type : avg
[2022/06/30 18:34:27] ppocr INFO:                 name : MobileNetV1Enhance
[2022/06/30 18:34:27] ppocr INFO:                 scale : 0.5
[2022/06/30 18:34:27] ppocr INFO:             Head : 
[2022/06/30 18:34:27] ppocr INFO:                 head_list : 
[2022/06/30 18:34:27] ppocr INFO:                     CTCHead : 
[2022/06/30 18:34:27] ppocr INFO:                         Head : 
[2022/06/30 18:34:27] ppocr INFO:                             fc_decay : 1e-05
[2022/06/30 18:34:27] ppocr INFO:                         Neck : 
[2022/06/30 18:34:27] ppocr INFO:                             depth : 2
[2022/06/30 18:34:27] ppocr INFO:                             dims : 64
[2022/06/30 18:34:27] ppocr INFO:                             hidden_dims : 120
[2022/06/30 18:34:27] ppocr INFO:                             name : svtr
[2022/06/30 18:34:27] ppocr INFO:                             use_guide : True
[2022/06/30 18:34:27] ppocr INFO:                     SARHead : 
[2022/06/30 18:34:27] ppocr INFO:                         enc_dim : 512
[2022/06/30 18:34:27] ppocr INFO:                         max_text_length : 25
[2022/06/30 18:34:27] ppocr INFO:                 name : MultiHead
[2022/06/30 18:34:27] ppocr INFO:             Transform : None
[2022/06/30 18:34:27] ppocr INFO:             algorithm : SVTR
[2022/06/30 18:34:27] ppocr INFO:             freeze_params : False
[2022/06/30 18:34:27] ppocr INFO:             model_type : rec
[2022/06/30 18:34:27] ppocr INFO:             pretrained : None
[2022/06/30 18:34:27] ppocr INFO:             return_all_feats : True
[2022/06/30 18:34:27] ppocr INFO:         Teacher : 
[2022/06/30 18:34:27] ppocr INFO:             Backbone : 
[2022/06/30 18:34:27] ppocr INFO:                 last_conv_stride : [1, 2]
[2022/06/30 18:34:27] ppocr INFO:                 last_pool_type : avg
[2022/06/30 18:34:27] ppocr INFO:                 name : MobileNetV1Enhance
[2022/06/30 18:34:27] ppocr INFO:                 scale : 0.5
[2022/06/30 18:34:27] ppocr INFO:             Head : 
[2022/06/30 18:34:27] ppocr INFO:                 head_list : 
[2022/06/30 18:34:27] ppocr INFO:                     CTCHead : 
[2022/06/30 18:34:27] ppocr INFO:                         Head : 
[2022/06/30 18:34:27] ppocr INFO:                             fc_decay : 1e-05
[2022/06/30 18:34:27] ppocr INFO:                         Neck : 
[2022/06/30 18:34:27] ppocr INFO:                             depth : 2
[2022/06/30 18:34:27] ppocr INFO:                             dims : 64
[2022/06/30 18:34:27] ppocr INFO:                             hidden_dims : 120
[2022/06/30 18:34:27] ppocr INFO:                             name : svtr
[2022/06/30 18:34:27] ppocr INFO:                             use_guide : True
[2022/06/30 18:34:27] ppocr INFO:                     SARHead : 
[2022/06/30 18:34:27] ppocr INFO:                         enc_dim : 512
[2022/06/30 18:34:27] ppocr INFO:                         max_text_length : 25
[2022/06/30 18:34:27] ppocr INFO:                 name : MultiHead
[2022/06/30 18:34:27] ppocr INFO:             Transform : None
[2022/06/30 18:34:27] ppocr INFO:             algorithm : SVTR
[2022/06/30 18:34:27] ppocr INFO:             freeze_params : False
[2022/06/30 18:34:27] ppocr INFO:             model_type : rec
[2022/06/30 18:34:27] ppocr INFO:             pretrained : None
[2022/06/30 18:34:27] ppocr INFO:             return_all_feats : True
[2022/06/30 18:34:27] ppocr INFO:     algorithm : Distillation
[2022/06/30 18:34:27] ppocr INFO:     model_type : rec
[2022/06/30 18:34:27] ppocr INFO:     name : DistillationModel
[2022/06/30 18:34:27] ppocr INFO: Eval : 
[2022/06/30 18:34:27] ppocr INFO:     dataset : 
[2022/06/30 18:34:27] ppocr INFO:         data_dir : ./total_text/test
[2022/06/30 18:34:27] ppocr INFO:         label_file_list : ['./total_text/test/test.txt']
[2022/06/30 18:34:27] ppocr INFO:         name : SimpleDataSet
[2022/06/30 18:34:27] ppocr INFO:         transforms : 
[2022/06/30 18:34:27] ppocr INFO:             DecodeImage : 
[2022/06/30 18:34:27] ppocr INFO:                 channel_first : False
[2022/06/30 18:34:27] ppocr INFO:                 img_mode : BGR
[2022/06/30 18:34:27] ppocr INFO:             MultiLabelEncode : None
[2022/06/30 18:34:27] ppocr INFO:             RecResizeImg : 
[2022/06/30 18:34:27] ppocr INFO:                 image_shape : [3, 48, 320]
[2022/06/30 18:34:27] ppocr INFO:             KeepKeys : 
[2022/06/30 18:34:27] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2022/06/30 18:34:27] ppocr INFO:     loader : 
[2022/06/30 18:34:27] ppocr INFO:         batch_size_per_card : 64
[2022/06/30 18:34:27] ppocr INFO:         drop_last : False
[2022/06/30 18:34:27] ppocr INFO:         num_workers : 4
[2022/06/30 18:34:27] ppocr INFO:         shuffle : False
[2022/06/30 18:34:27] ppocr INFO: Global : 
[2022/06/30 18:34:27] ppocr INFO:     cal_metric_during_train : True
[2022/06/30 18:34:27] ppocr INFO:     character_dict_path : ppocr/utils/ppocr_keys_v1.txt
[2022/06/30 18:34:27] ppocr INFO:     checkpoints : None
[2022/06/30 18:34:27] ppocr INFO:     debug : False
[2022/06/30 18:34:27] ppocr INFO:     distributed : False
[2022/06/30 18:34:27] ppocr INFO:     epoch_num : 800
[2022/06/30 18:34:27] ppocr INFO:     eval_batch_step : [0, 2000]
[2022/06/30 18:34:27] ppocr INFO:     infer_img : doc/imgs_words/ch/word_1.jpg
[2022/06/30 18:34:27] ppocr INFO:     infer_mode : False
[2022/06/30 18:34:27] ppocr INFO:     log_smooth_window : 20
[2022/06/30 18:34:27] ppocr INFO:     max_text_length : 25
[2022/06/30 18:34:27] ppocr INFO:     pretrained_model : ./pretrain_model/ch_PP-OCRv3_rec_train/best_accuracy
[2022/06/30 18:34:27] ppocr INFO:     print_batch_step : 10
[2022/06/30 18:34:27] ppocr INFO:     save_epoch_step : 3
[2022/06/30 18:34:27] ppocr INFO:     save_inference_dir : None
[2022/06/30 18:34:27] ppocr INFO:     save_model_dir : ./output/rec_ppocr_v3_distillation_test_v1.0
[2022/06/30 18:34:27] ppocr INFO:     save_res_path : ./output/rec/predicts_ppocrv3_distillation.txt
[2022/06/30 18:34:27] ppocr INFO:     use_gpu : True
[2022/06/30 18:34:27] ppocr INFO:     use_space_char : True
[2022/06/30 18:34:27] ppocr INFO:     use_visualdl : False
[2022/06/30 18:34:27] ppocr INFO: Loss : 
[2022/06/30 18:34:27] ppocr INFO:     loss_config_list : 
[2022/06/30 18:34:27] ppocr INFO:         DistillationDMLLoss : 
[2022/06/30 18:34:27] ppocr INFO:             act : softmax
[2022/06/30 18:34:27] ppocr INFO:             dis_head : ctc
[2022/06/30 18:34:27] ppocr INFO:             key : head_out
[2022/06/30 18:34:27] ppocr INFO:             model_name_pairs : [['Student', 'Teacher']]
[2022/06/30 18:34:27] ppocr INFO:             multi_head : True
[2022/06/30 18:34:27] ppocr INFO:             name : dml_ctc
[2022/06/30 18:34:27] ppocr INFO:             use_log : True
[2022/06/30 18:34:27] ppocr INFO:             weight : 1.0
[2022/06/30 18:34:27] ppocr INFO:         DistillationDMLLoss : 
[2022/06/30 18:34:27] ppocr INFO:             act : softmax
[2022/06/30 18:34:27] ppocr INFO:             dis_head : sar
[2022/06/30 18:34:27] ppocr INFO:             key : head_out
[2022/06/30 18:34:27] ppocr INFO:             model_name_pairs : [['Student', 'Teacher']]
[2022/06/30 18:34:27] ppocr INFO:             multi_head : True
[2022/06/30 18:34:27] ppocr INFO:             name : dml_sar
[2022/06/30 18:34:27] ppocr INFO:             use_log : True
[2022/06/30 18:34:27] ppocr INFO:             weight : 0.5
[2022/06/30 18:34:27] ppocr INFO:         DistillationDistanceLoss : 
[2022/06/30 18:34:27] ppocr INFO:             key : backbone_out
[2022/06/30 18:34:27] ppocr INFO:             mode : l2
[2022/06/30 18:34:27] ppocr INFO:             model_name_pairs : [['Student', 'Teacher']]
[2022/06/30 18:34:27] ppocr INFO:             weight : 1.0
[2022/06/30 18:34:27] ppocr INFO:         DistillationCTCLoss : 
[2022/06/30 18:34:27] ppocr INFO:             key : head_out
[2022/06/30 18:34:27] ppocr INFO:             model_name_list : ['Student', 'Teacher']
[2022/06/30 18:34:27] ppocr INFO:             multi_head : True
[2022/06/30 18:34:27] ppocr INFO:             weight : 1.0
[2022/06/30 18:34:27] ppocr INFO:         DistillationSARLoss : 
[2022/06/30 18:34:27] ppocr INFO:             key : head_out
[2022/06/30 18:34:27] ppocr INFO:             model_name_list : ['Student', 'Teacher']
[2022/06/30 18:34:27] ppocr INFO:             multi_head : True
[2022/06/30 18:34:27] ppocr INFO:             weight : 1.0
[2022/06/30 18:34:27] ppocr INFO:     name : CombinedLoss
[2022/06/30 18:34:27] ppocr INFO: Metric : 
[2022/06/30 18:34:27] ppocr INFO:     base_metric_name : RecMetric
[2022/06/30 18:34:27] ppocr INFO:     ignore_space : False
[2022/06/30 18:34:27] ppocr INFO:     key : Student
[2022/06/30 18:34:27] ppocr INFO:     main_indicator : acc
[2022/06/30 18:34:27] ppocr INFO:     name : DistillationMetric
[2022/06/30 18:34:27] ppocr INFO: Optimizer : 
[2022/06/30 18:34:27] ppocr INFO:     beta1 : 0.9
[2022/06/30 18:34:27] ppocr INFO:     beta2 : 0.999
[2022/06/30 18:34:27] ppocr INFO:     lr : 
[2022/06/30 18:34:27] ppocr INFO:         decay_epochs : [700, 800]
[2022/06/30 18:34:27] ppocr INFO:         name : Piecewise
[2022/06/30 18:34:27] ppocr INFO:         values : [0.0005, 5e-05]
[2022/06/30 18:34:27] ppocr INFO:         warmup_epoch : 5
[2022/06/30 18:34:27] ppocr INFO:     name : Adam
[2022/06/30 18:34:27] ppocr INFO:     regularizer : 
[2022/06/30 18:34:27] ppocr INFO:         factor : 3e-05
[2022/06/30 18:34:27] ppocr INFO:         name : L2
[2022/06/30 18:34:27] ppocr INFO: PostProcess : 
[2022/06/30 18:34:27] ppocr INFO:     key : head_out
[2022/06/30 18:34:27] ppocr INFO:     model_name : ['Student', 'Teacher']
[2022/06/30 18:34:27] ppocr INFO:     multi_head : True
[2022/06/30 18:34:27] ppocr INFO:     name : DistillationCTCLabelDecode
[2022/06/30 18:34:27] ppocr INFO: Train : 
[2022/06/30 18:34:27] ppocr INFO:     dataset : 
[2022/06/30 18:34:27] ppocr INFO:         data_dir : ./total_text/train
[2022/06/30 18:34:27] ppocr INFO:         ext_op_transform_idx : 1
[2022/06/30 18:34:27] ppocr INFO:         label_file_list : ['./total_text/train/train.txt']
[2022/06/30 18:34:27] ppocr INFO:         name : SimpleDataSet
[2022/06/30 18:34:27] ppocr INFO:         transforms : 
[2022/06/30 18:34:27] ppocr INFO:             DecodeImage : 
[2022/06/30 18:34:27] ppocr INFO:                 channel_first : False
[2022/06/30 18:34:27] ppocr INFO:                 img_mode : BGR
[2022/06/30 18:34:27] ppocr INFO:             RecConAug : 
[2022/06/30 18:34:27] ppocr INFO:                 ext_data_num : 2
[2022/06/30 18:34:27] ppocr INFO:                 image_shape : [48, 320, 3]
[2022/06/30 18:34:27] ppocr INFO:                 prob : 0.5
[2022/06/30 18:34:27] ppocr INFO:             RecAug : None
[2022/06/30 18:34:27] ppocr INFO:             MultiLabelEncode : None
[2022/06/30 18:34:27] ppocr INFO:             RecResizeImg : 
[2022/06/30 18:34:27] ppocr INFO:                 image_shape : [3, 48, 320]
[2022/06/30 18:34:27] ppocr INFO:             KeepKeys : 
[2022/06/30 18:34:27] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2022/06/30 18:34:27] ppocr INFO:     loader : 
[2022/06/30 18:34:27] ppocr INFO:         batch_size_per_card : 32
[2022/06/30 18:34:27] ppocr INFO:         drop_last : True
[2022/06/30 18:34:27] ppocr INFO:         num_workers : 4
[2022/06/30 18:34:27] ppocr INFO:         shuffle : True
[2022/06/30 18:34:27] ppocr INFO: profiler_options : None
[2022/06/30 18:34:27] ppocr INFO: train with paddle 2.2.2 and device CUDAPlace(0)
[2022/06/30 18:34:27] ppocr INFO: Initialize indexs of datasets:['./total_text/train/train.txt']
[2022/06/30 18:34:27] ppocr INFO: Initialize indexs of datasets:['./total_text/test/test.txt']
[2022/06/30 18:34:33] ppocr INFO: load pretrain successful from ./pretrain_model/ch_PP-OCRv3_rec_train/best_accuracy
[2022/06/30 18:34:33] ppocr INFO: train dataloader has 39 iters
[2022/06/30 18:34:33] ppocr INFO: valid dataloader has 5 iters
[2022/06/30 18:34:33] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations

**Comment by asus10tu** [comment]
需要补充什么资料么，至今还是没能解决，求帮忙~

**Comment by tink2123** [comment]
训练batch_size减小至1，试下能不能跑起来。

**Comment by asus10tu** [comment]
> 训练batch_size减小至1，试下能不能跑起来。

你好，batch_size减小至1，num_workers也减小到1，情况依旧是自动停止了，请问还有什么其他可以调整或者可以追溯日志的地方呢，谢谢

**Comment by selfshore** [comment]
我也遇到这个问题，请问解决了吗？

**Comment by sunzhaoyang** [comment]
同样的问题，蹲一个

**Comment by Louis24** [comment]
Thank you 问题解决 另外求图像算法内推

**Comment by taojing7** [comment]
之前遇到这个问题是显存爆了，调到最低可以跑，8g显存

**Comment by LeoRainly** [comment]
我也遇到这个问题，启动训练后训练了十几个epoch就自动停掉了，显存还占着
